instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py

```diff
@@ -51,6 +53,46 @@
 ]
 
 
+class _ExcludeList:
```

Comment:
This code would be useful to share across instrumentations. This was discussed at the Jan 14th SIG meeting, and everyone present agreed it could be moved into the `opentelemetry-instrumentation` package

Reply:
Before moving this into `opentelemetry-instrumentation` is there anything else you want me to move there as well? I see how also the common code that is used to handle URLs can be placed there.